src/app/main/component/user/components/profile/user-notifications/user-notifications-pop-up/user-notifications-pop-up.component.html

```diff
@@ -0,0 +1,38 @@
+<div
+  [ngClass]="{ 'pop-up-container': notificationToShow.length, 'pop-up-container-empty': !notificationToShow.length }"
```

Comment:
You can use here `"notificationToShow.length ? 'pop-up-container' : 'pop-up-container-empty'"`